CONTRIBUTING.md

```
@@ -92,19 +93,19 @@ pnpm dev
 
 ## Component Development
 
-1. Create components in `packages/raystack/`
+1. Create components in `packages/raystack/components/`
 2. Follow the existing component structure:
    ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)